docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -356,6 +356,8 @@ Standard ONNX\* operators:
 | ROIAlign | No |
 | Range | No |
 | Reciprocal | No |
+| ReduceL1 | No |
```

Comment:
How about other frameworks? Do they have ReduceLx operations?

Reply:
TensorFlow have `reduce_euclidean_norm` operation same as ReduceL2 operation from ONNX , extractor for `reduce_euclidean_norm` added.